### ISSUE TITLE
Enhancement Request

### ISSUE THREAD

**Issue by sydbat** [byoso/Nord-Manager#11]
Is it possible to add a check mark next to the active server? Or possibly highlight the active server?
Otherwise, so far, this app seems to work well. Thank you for your efforts.

**Comment by byoso** [comment]
Actually, when using the option "browse...", the window appearing is dynamically built from the datas given by nord vpn, only active servers are supposed to be there. So I assume that you are talking about the custom buttons, and, well, it is not a bad idea, I'll try to think about that the next time I work on nord manager ;).

Anyway, thank you a lot for your return and suggestion.

**Comment by sydbat** [comment]
Yes, the custom buttons. That would make this perfect. Thank you for responding.
